staging/src/k8s.io/apimachinery/pkg/util/sets/set_generic_test.go

```diff
@@ -0,0 +1,305 @@
+/*
```

Comment:
We should make a benchmark comparing with the current types, because these are used all over.

Reply:
Yes, I will add benchmark test after making things right.